### PR TITLE
Fix rst syntax error in README.rst

### PR DESCRIPTION
**Pull request by hf-krechan**
One rst syntax error stopped the release process.
To avoid this error in the future, I added a new tox environment to test the packaging process.
All steps to release a new bo4e package version are now described in the README.rst

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -25,11 +25,27 @@ Feel free to open a Pull Request against the develop branch of this repository.
...
+
    tox -e dev 
    
 The created venv should be located somewhere around .tox/dev/Scripts.
 
+Release workflow
+================
+* Check with tox all tests and lintings: `tox`
+* Check with tox if the packaging works fine: `tox -e test_packaging`
+* Merge all your changes you would like to have in the release into the master branch.
```
Comment (hf-kklein) [inline README.rst:38]:
```suggestion
* Merge all your changes you would like to have in the release into the master branch
```

File: README.rst [hunk 2]
```diff
@@ -25,11 +25,27 @@ Feel free to open a Pull Request against the develop branch of this repository.
...
+* Check that all github actions for tests and linting do pass.
```
Comment (hf-kklein) [inline README.rst:39]:
```suggestion
* Check that all Github actions for tests and linting do pass (should be automatically enforced for PRs against master) 
```

File: README.rst [hunk 3]
```diff
@@ -25,11 +25,27 @@ Feel free to open a Pull Request against the develop branch of this repository.
...
+* Go to `BO4E-python`_ and click on *Create a new release* in the right sidebar
```
Comment (hf-kklein) [inline README.rst:40]: you can create a link to that form directly?

File: README.rst [hunk 4]
```diff
@@ -25,11 +25,27 @@ Feel free to open a Pull Request against the develop branch of this repository.
...
+* Write in the *Tag version* field and in the *Release title* your new version, i.e. `v0.0.6`
+* Add a describtion to the release.
```
Comment (hf-kklein) [inline README.rst:42]:
```suggestion
* Add a describtion to the release
```

File: README.rst [hunk 5]
```diff
@@ -25,11 +25,27 @@ Feel free to open a Pull Request against the develop branch of this repository.
...
+* Publish the release
+
+There is a github action which gets triggered by a release event.
+It will run all default tests with tox. If they pass, it will take the tag title to replace the version information in the *setup.cfg* file.
+After checking the package with `twize check` it will finally upload the new package release.
```
Comment (hf-kklein) [inline README.rst:47]: twize oder twine?

**Review by hf-kklein — APPROVED** [review]
minor consistency stuff

File: README.rst (hunk 5 above)
Comment (hf-krechan) [reply README.rst:47]: Natürlich `twine`. Danke

File: README.rst (hunk 3 above)
Comment (hf-krechan) [reply README.rst:40]: No, I added the link at the bottom of the README.

Comment (hf-kklein) [reply README.rst:40]: I thought of this link: https://github.com/Hochfrequenz/BO4E-python/releases/new

**Pull request merged**